### PR TITLE
[JENKINS-69320] Resize behavior of Execute Shell build steps is broken

### PR DESCRIPTION
**Pull request by benebsiny**
See [JENKINS-69320](https://issues.jenkins.io/browse/JENKINS-69320).

https://user-images.githubusercontent.com/20413055/184808876-6914fba0-e16b-4e52-816d-a48a2307f2af.mp4

### Proposed changelog entries

* Fix the resize behavior of Execute Shell build steps

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6999"></a>

### CODE REVIEW

**Comment by daniel-beck** [comment]
Thanks for taking care of this!

File: war/src/main/webapp/scripts/hudson-behavior.js
Comment (daniel-beck) [inline war/src/main/webapp/scripts/hudson-behavior.js:1180]: This can be a problem for input fields like Execute Shell, whose default size changes based on the size of the script content. Maybe some folks would like to make this smaller temporarily, but can't:

> Screenshot 2022-08-16 at 12 44 32

Probably

```suggestion
      const MIN_HEIGHT = Math.min(200, codemirror.clientHeight);
```

or so?

**Review by daniel-beck — CHANGES_REQUESTED** [review]

File: war/src/main/webapp/scripts/hudson-behavior.js
Comment (benebsiny) [reply war/src/main/webapp/scripts/hudson-behavior.js]: Yess. With this, user can reduce the height of fields to 200px. Thanks!

**Review by daniel-beck — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
